src/ansys/speos/core/source.py

```diff
@@ -418,6 +418,7 @@ class SourceLuminaire(BaseSource):
         Uses default values when True.
     """
 
+    @general_methods.min_speos_version(25, 2, 0)
```

Comment:
should warning be added in the __init__ or at the function?

Reply:
when we add it at function the function itself looses it type -> we need to do it at init

Reply:
see previous iterration with lots of failed unit tests